packages/meditrak-server/src/utilities/sendEmail.js

```diff
@@ -5,7 +5,7 @@
 import nodemailer from 'nodemailer';
 import { getIsProductionEnvironment } from '../devops';
 
-export const sendEmail = (to, subject, text) => {
+export const sendEmail = (to, subject, text, attachments) => {
```

Comment:
(Not required, but helpful for devs)

```suggestion
export const sendEmail = async (to, subject, text, attachments) => {
```